clients/client-python/requirements.txt

```diff
@@ -1,4 +0,0 @@
-# Copyright 2024 Datastrato Pvt Ltd.
-# This software is licensed under the Apache License version 2.
-pytest~=8.0.1
```

Comment:
I think we should keep this file. In https://github.com/datastrato/gravitino/pull/2684, I use `requirements.txt` to install dependencies. If this changed, we also need to update the gradle configuration. What do you think? @zhaoyongjie

Reply:
@coolderli the dependencies was moved in `setup.py` and `requests` was moved away in recent commit, so if you wanna install current project with dev-dependencies run command below

```shell
pip install -e .[dev]
```

if the normal mode installation needed, run command below

```
pip install -e .
```

Furthermore,  if the project were published to PyPI, the following would be supported:

```
pip install gravitino
```